build-tests-samples/heft-serverless-stack-tutorial/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+
+  "compilerOptions": {
+    "outDir": "lib",
+    "rootDir": "src",
+
+    "forceConsistentCasingInFileNames": true,
```

Comment:
We should set `"strict": true` in the test and example projexts.

Reply:
These sample files drifted out of sync with the rigs. I've updated **tsconfig.json** across all the sample projects.